bugbug/tools/code_review.py

```diff
@@ -1043,44 +1043,54 @@ def __init__(
             else ""
         )
 
-        self.summarization_chain = LLMChain(
-            prompt=PromptTemplate.from_template(
-                PROMPT_TEMPLATE_SUMMARIZATION,
-                partial_variables={
-                    "experience_scope": (
-                        f"the {self.target_software} source code"
-                        if self.target_software
-                        else "a software project"
-                    )
-                },
-            ),
-            llm=self.llm,
-            verbose=verbose,
+        summarization_chain_prompt = PromptTemplate.from_template(
+            PROMPT_TEMPLATE_SUMMARIZATION,
+            partial_variables={
+                "experience_scope": (
+                    f"the {self.target_software} source code"
+                    if self.target_software
+                    else "a software project"
+                )
+            },
         )
-        self.filtering_chain = LLMChain(
-            prompt=PromptTemplate.from_template(
-                PROMPT_TEMPLATE_FILTERING_ANALYSIS,
-                partial_variables={
-                    "target_code_consistency": self.target_software or "rest of the"
-                },
-            ),
-            llm=self.llm,
-            verbose=verbose,
+
+        self.summarization_chain = RunnableSequence(
+            [summarization_chain_prompt, self.llm], verbose=verbose
+        )
+
+        filtering_chain_prompt = PromptTemplate.from_template(
+            PROMPT_TEMPLATE_FILTERING_ANALYSIS,
+            partial_variables={
+                "target_code_consistency": self.target_software or "rest of the"
+            },
+        )
+
+        self.filtering_chain = RunnableSequence(
+            [filtering_chain_prompt, self.llm], verbose=verbose
+        )
+
+        deduplicating_chain_prompt = PromptTemplate.from_template(
+            PROMPT_TEMPLATE_DEDUPLICATE
+        )
+
+        self.deduplicating_chain = RunnableSequence(
+            [deduplicating_chain_prompt, self.llm], verbose=verbose
         )
-        self.deduplicating_chain = LLMChain(
-            prompt=PromptTemplate.from_template(PROMPT_TEMPLATE_DEDUPLICATE),
-            llm=self.llm,
-            verbose=verbose,
+
+        further_context_prompt = PromptTemplate.from_template(
+            PROMPT_TEMPLATE_FURTHER_CONTEXT_LINES
         )
-        self.further_context_chain = LLMChain(
-            prompt=PromptTemplate.from_template(PROMPT_TEMPLATE_FURTHER_CONTEXT_LINES),
-            llm=self.llm,
-            verbose=verbose,
+
+        self.further_context_chain = RunnableSequence(
+            [further_context_prompt, self.llm], verbose=verbose
```

Comment:
can verbose used here?

Reply:
I don't think so; you could check the documentation and test it.